### PR TITLE
Favorite event

### PR DESCRIPTION
**Pull request by yuda1124**
Add favorite event function.
And now, you can see your favorite event in my page view!

### CODE REVIEW

File: shared/components/Card.js [hunk 1]
```diff
@@ -115,10 +116,50 @@ class Card extends Component {
...
   handlePressStar() {
-    if (this.state.starClicked === true) {
-      this.setState({starClicked: false});
+    if (this.state.isSaved === true) {
+      this.setState({isSaved: false});
+      const address = `${HTTPS}${SERVER_ADDR}${ENDPOINT_SAVEDPOST}`;
+      const bodySave = JSON.stringify({
+        'itemKey': `${this.props.dataSource.key}`
+      });
+      console.log("yaya" + address);
```
Comment (ochanje210) [inline shared/components/Card.js]: console log

File: shared/components/myPage.js [hunk 2]
```diff
@@ -226,6 +227,7 @@ class MyPage extends Component {
   }
 
   render() {
+    console.log(this.props.savedPosts);
```
Comment (ochanje210) [inline shared/components/myPage.js]: console.log

File: shared/components/eventReportView.js [hunk 3]
```diff
@@ -59,9 +60,10 @@ export default class EventReportView extends Component {
         return response.json();
       })
       .then((rjson)=>{
-        console.log(rjson);
-        (this.props.aboutPhoto) ? this.props.handleReport('photo') : this.props.handleReport('location');
-        Actions.pop();
+        if (rjson.message === 'success') {
```
Comment (yuda1124) [inline shared/components/eventReportView.js]: good point. I'll fix it

**Comment by ochanje210** [comment]
shipit

**Comment by ochanje210** [comment]
lgtm

**Pull request merged**